### PR TITLE
Add polygon, bsc

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

**Review by neewy — DISMISSED** [review]

**Review by easeev — DISMISSED** [review]
LGTM

File: docs/platform/join-a-public-network.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 meta:
   - name: description
-    content: Learn how to deploy a node on Ethereum, Bitcoin, or Corda in minutes with the Chainstack managed blockchain services.
+    content: Learn how to deploy a node on Ethereum, Polygon PoS, Binance Smart Chain, Bitcoin, or Corda in minutes with the Chainstack managed blockchain services.
   - name: keywords
     content: ethereum corda bitcoin deploy node
```
Comment (easeev) [inline docs/platform/join-a-public-network.md:6]: Perhaps keywords need to be updated as well

File: docs/platform/join-a-public-network.md [hunk 2]
```diff
@@ -25,12 +25,13 @@ You can currently join the public Ethereum, Bitcoin, and Corda networks.
 
 The node status will change from **Pending** to **Running** once deployed.
 
-## Join a Bitcoin network
+## Join a Polygon PoS network
 
 1. Select a [public chain project](/glossary/public-chain-project) and click **Get started** or **Join network**.
-1. Under **Blockchain protocol**, select **Bitcoin**.
+1. Under **Blockchain protocol**, select **Polygon PoS**.
 1. Under **Blockchain network**, select **Mainnet** or **Testnet**. Click **Next**.
```
Comment (easeev) [inline docs/platform/join-a-public-network.md:32]: @CaJIbHuK did we change `Testnet` to `Mumbai testnet` in the UI for Polygon as discussed?

https://chainstackhq.slack.com/archives/CBN056831/p1620730850051200

Comment (CaJIbHuK) [reply docs/platform/join-a-public-network.md:32]: Hm, for some reason UI doesn't fully reflect the changes. Selector in wizard is left untouched, I'll check on Mon and probably release within the next small tech release

image
image
image

Comment (CaJIbHuK) [reply docs/platform/join-a-public-network.md:32]: https://github.com/chainstack/ui/pull/2340

File: docs/platform/join-a-public-network.md (hunk 1 above)
Comment (akegaviar) [reply docs/platform/join-a-public-network.md]: Done

File: docs/platform/join-a-public-network.md (hunk 2 above)
Comment (akegaviar) [reply docs/platform/join-a-public-network.md]: Done

**Comment by CaJIbHuK** [comment]
@easeev pls review

Shall I released a new version once it's merged

**Comment by easeev** [comment]
> Shall I release a new version once it's merged

Not urgent, can wait until the next release

**Review by easeev — APPROVED** [review]
LGTM

**Review by neewy — APPROVED** [review]

**Pull request merged**